docs/guides/qiskit-addons.mdx

```diff
@@ -5,7 +5,7 @@ description: Understand the Qiskit addon tools , which help you build utility-gr
 
 # Qiskit addons
 
-Qiskit addons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](/intro-to-patterns)
+Qiskit addons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](intro-to-patterns)
```

Comment:
Do you mean this? Alternatively it could be /guides/intro-to-patterns

```suggestion
Qiskit addons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](./intro-to-patterns)
```

Reply:
If you're linking from a file in guides to another file in guides, why would you want /guides?

Reply:
It doesn't matter which of the two styles you use. They both behave the same. So whatever you prefer